modules/rosa-upgrade-cluster-cli.adoc

```diff
@@ -101,3 +101,95 @@ $ rosa delete upgrade --cluster=<cluster_name> | <cluster_id>
 |--yes
 |Automatically answers `yes` to confirm the operation.
 |===
+
+[id="rosa-upgrade-machinepool_{context}"]
+== upgrade machinepool
+
+Upgrades a specific machine pool configured on a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa upgrade machinepool --cluster=<cluster_name> --machinepool=<machinepool_name>| <cluster_id> <machinepool_id> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|Required: The name or ID (string) of the cluster.
+
+|--machinepool
+|Required: The name or ID (string) of the machinepool.
+
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--profile
+|Specifies an AWS profile (string) from your credentials file.
+|===
+
+.Example
+Upgrade a machine pool named `mymachinepool` on a cluster named `mycluster`.
+[source,terminal]
+----
+$ rosa upgrade machinepool --cluster=mycluster --machinepool=mymachinepool
```

Comment:
It is not correct. It should like this
`rosa upgrade machinepool --cluster=mycluster --machinepool=mymachinepool`